Adjust the test expectation for stdio-common/tst-grouping3 to match
the state of the RHEL 9 printf implementation, which does not add
zero padding when grouping integers.

diff --git a/stdio-common/tst-grouping3.c b/stdio-common/tst-grouping3.c
index e9e39218e25a2720..5a247cfe3fb8564f 100644
--- a/stdio-common/tst-grouping3.c
+++ b/stdio-common/tst-grouping3.c
@@ -46,7 +46,7 @@ do_test (void)
      This bug is distinct from bug 23432 which has to do with the
      minimum precision calculation (digit based).  */
   sprintf (buf, "%+-'13.9d", 1234567);
-  TEST_COMPARE_STRING (buf, "+001.234.567 ");
+  TEST_COMPARE_STRING (buf, "+1.234.567   ");
 
   return 0;
 }
